netlify/netlify-plugin-nextjs
# chore: add e2e tests and demos

### Summary
Add demos and e2e tests to the default site for a bunch of next features.

Namely: 
- Environment tokens
- Font handling
- Routing
- CSS handling
- Layouts
- TypeScript
- Scripts
- Images
- Middleware

### Test plan

1. Visit the Default Deploy Preview (`/` should work even though it's typescript)
2. Go to `/css`
3. Go to `/script`
4. Go to `/layouts`
5. Go to `/env`
6. Go to `/font`
7. Go to `/broken-image` (expected broken page)
8. Go to `/middle`
8. See the default demo cypress test run

Example new demo page:
<img width="775" alt="CleanShot 2022-01-27 at 20 08 38@2x" src="https://user-images.githubusercontent.com/11183523/151408341-2eb6f534-5db7-4d77-b097-9b18545cc587.png">


### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal

### Standard checks:




---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.
